inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/SinkType.java

```diff
@@ -39,4 +47,27 @@ public class SinkType extends StreamType {
      * Details: <a href="https://www.tencentcloud.com/products/cls">CLS</a>
      */
     public static final String CLS = "CLS";
+
+    public static final Set<String> SORT_FLINK_SINK = new HashSet<>();
+
+    public static final Set<String> SORT_STANDALONE_SINK = new HashSet<>();
+
+    public static boolean containSortFlinkSink(List<String> sinkTypes) {
```

Comment:
use  Set::retainAll to get the intersection of two sets

Reply:
Done.